src/main/java/net/sf/mzmine/main/MZmineCore.java

```diff
@@ -220,6 +220,10 @@ public void run() {
 	    NewVersionCheck NVC = new NewVersionCheck(CheckType.DESKTOP);
 	    new Thread(NVC).start();
 
+	    // Tracker
+	    GoogleAnalyticsTracker GAT = new GoogleAnalyticsTracker("GUI Loaded", "/JAVA/Main/GUI");
+	    new Thread(GAT).start();
```

Comment:
The Thread class has a function setPriority().
I think it could be a good idea to set the priority of these two threads (GAT and NVC) to the lowest possible value.